bin/node/src/chain_spec.rs

```diff
@@ -282,23 +316,29 @@ fn genesis(
     // NOTE: some combinations of bootstrap chain arguments can potentially
     // lead to duplicated rich accounts, e.g. if a sudo account is also an authority
     // which is why we remove the duplicates if any here
+    // endow as well stash accounts, if any
     let unique_accounts: Vec<AccountId> = deduplicate(
-        authorities
-            .iter()
-            .map(|auth| &auth.account_id)
-            .cloned()
+        to_account_ids(&authorities)
             .chain(special_accounts)
+            .chain(stakers.iter().cloned())
             .collect(),
     );
 
+    const ENDOWMENT: u128 = 1 << 60;
+    const MIN_VALIDATOR_BOND: u128 = 25_000 * 1_000_000_000_000;
```

Comment:
Would be good to have `const AZERO = ....` and then represent other values in terms of `AZERO`.

Reply:
Fixed.